components/tabs/Tab.js

```diff
@@ -40,6 +42,18 @@ class Tab extends Component {
     }
   };
 
+  renderIcon = () => {
+    if (typeof this.props.icon === 'undefined') {
+      return null;
+    } else if (typeof this.props.icon === 'string') {
+      return (
```

Comment:
The div is needed to ensure the tab icon and the tab label appear on separate lines as required by the spec (without it they appear side-by-side).

You are right about the string handling in FontIcon, so I fixed that.